packages/loot-core/webpack/webpack.desktop.config.js

```diff
@@ -24,14 +24,16 @@ module.exports = {
       '.ts',
       '.tsx',
       '.json',
+      'pegjs',
```

Comment:
Why was this added?

Reply:
It was the only way i could get it to build - goal templates use some .pegjs files that I guess were not getting included in the webpack bundle file. 
If you see [this](https://github.com/actualbudget/actual/pull/988/commits/73a4c55c6ecddc37bd7d98f34b193448cc99e4af) commit by Matiss you can see a to-do item referencing this issue.